[2022-11-07T23:26:56.656-0300] {processor.py:156} INFO - Started process (PID=965181) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:26:56.657-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:26:56.657-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:26:56.657-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:26:56.662-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:26:56.660-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:26:56.663-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:26:56.684-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.031 seconds
[2022-11-07T23:27:36.676-0300] {processor.py:156} INFO - Started process (PID=965400) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:27:36.680-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:27:36.681-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:27:36.681-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:27:36.686-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:27:36.685-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:27:36.686-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:27:36.707-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.038 seconds
[2022-11-07T23:28:13.703-0300] {processor.py:156} INFO - Started process (PID=965475) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:28:13.706-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:28:13.707-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:28:13.707-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:28:13.713-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:28:13.712-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:28:13.714-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:28:13.745-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.046 seconds
[2022-11-07T23:28:50.447-0300] {processor.py:156} INFO - Started process (PID=965552) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:28:50.450-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:28:50.450-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:28:50.450-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:28:50.454-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:28:50.453-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:28:50.454-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:28:50.503-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.063 seconds
[2022-11-07T23:29:26.622-0300] {processor.py:156} INFO - Started process (PID=965623) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:29:26.624-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:29:26.625-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:29:26.625-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:29:26.629-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:29:26.628-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:29:26.629-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:29:26.653-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.033 seconds
[2022-11-07T23:29:57.219-0300] {processor.py:156} INFO - Started process (PID=965684) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:29:57.220-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:29:57.221-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:29:57.221-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:29:57.227-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:29:57.226-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:29:57.228-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:29:57.249-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.034 seconds
[2022-11-07T23:30:32.644-0300] {processor.py:156} INFO - Started process (PID=965756) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:30:32.646-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:30:32.647-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:30:32.647-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:30:32.650-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:30:32.649-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:30:32.651-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:30:32.673-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.031 seconds
[2022-11-07T23:31:09.509-0300] {processor.py:156} INFO - Started process (PID=965828) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:31:09.511-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:31:09.512-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:31:09.512-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:31:09.517-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:31:09.516-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:31:09.518-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:31:09.541-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.035 seconds
[2022-11-07T23:31:43.550-0300] {processor.py:156} INFO - Started process (PID=965894) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:31:43.553-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:31:43.554-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:31:43.554-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:31:43.561-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:31:43.559-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:31:43.562-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:31:43.592-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.050 seconds
[2022-11-07T23:32:22.087-0300] {processor.py:156} INFO - Started process (PID=966103) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:32:22.091-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:32:22.093-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:32:22.092-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:32:22.098-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:32:22.097-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:32:22.098-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:32:22.123-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.043 seconds
[2022-11-07T23:32:52.493-0300] {processor.py:156} INFO - Started process (PID=966163) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:32:52.495-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:32:52.496-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:32:52.496-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:32:52.500-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:32:52.499-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:32:52.501-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:32:52.526-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.036 seconds
[2022-11-07T23:33:29.727-0300] {processor.py:156} INFO - Started process (PID=966237) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:33:29.730-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:33:29.731-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:33:29.731-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:33:29.740-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:33:29.738-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:33:29.741-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:33:29.769-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.049 seconds
[2022-11-07T23:34:07.329-0300] {processor.py:156} INFO - Started process (PID=966311) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:34:07.332-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:34:07.333-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:34:07.333-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:34:07.337-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:34:07.336-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:34:07.337-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:34:07.359-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.035 seconds
[2022-11-07T23:34:44.417-0300] {processor.py:156} INFO - Started process (PID=966387) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:34:44.419-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:34:44.420-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:34:44.420-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:34:44.423-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:34:44.422-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:34:44.424-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:34:44.447-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.034 seconds
[2022-11-07T23:35:15.117-0300] {processor.py:156} INFO - Started process (PID=966448) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:35:15.119-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:35:15.120-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:35:15.119-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:35:15.125-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:35:15.124-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:35:15.126-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:35:15.150-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.038 seconds
[2022-11-07T23:35:46.935-0300] {processor.py:156} INFO - Started process (PID=966512) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:35:46.938-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:35:46.940-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:35:46.939-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:35:46.947-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:35:46.946-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:35:46.948-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:35:46.982-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.053 seconds
[2022-11-07T23:36:27.769-0300] {processor.py:156} INFO - Started process (PID=966591) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:36:27.772-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:36:27.773-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:36:27.773-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:36:27.777-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:36:27.776-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:36:27.778-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:36:27.804-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.039 seconds
[2022-11-07T23:37:06.174-0300] {processor.py:156} INFO - Started process (PID=966669) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:37:06.177-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:37:06.178-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:37:06.178-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:37:06.184-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:37:06.183-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:37:06.185-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:37:06.212-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.043 seconds
[2022-11-07T23:37:41.888-0300] {processor.py:156} INFO - Started process (PID=966869) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:37:41.913-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:37:41.914-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:37:41.913-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:37:41.920-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:37:41.919-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:37:41.921-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:37:41.955-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.084 seconds
[2022-11-07T23:38:17.554-0300] {processor.py:156} INFO - Started process (PID=966939) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:38:17.557-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:38:17.557-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:38:17.557-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:38:17.561-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:38:17.560-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:38:17.561-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:38:17.585-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.034 seconds
[2022-11-07T23:38:47.804-0300] {processor.py:156} INFO - Started process (PID=966997) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:38:47.805-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:38:47.806-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:38:47.806-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:38:47.810-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:38:47.809-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 4, in <module>
    from airflow.operators.alura import TwitterOperator
ModuleNotFoundError: No module named 'airflow.operators.alura'
[2022-11-07T23:38:47.810-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:38:47.832-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.032 seconds
[2022-11-07T23:39:18.740-0300] {processor.py:156} INFO - Started process (PID=967077) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:39:18.742-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:39:18.743-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:39:18.742-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:39:18.764-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:39:18.873-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:39:18.873-0300] {manager.py:507} INFO - Created Permission View: can edit on DAG:twitter_dag
[2022-11-07T23:39:18.892-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:39:18.892-0300] {manager.py:507} INFO - Created Permission View: can read on DAG:twitter_dag
[2022-11-07T23:39:18.906-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:39:18.905-0300] {manager.py:507} INFO - Created Permission View: can delete on DAG:twitter_dag
[2022-11-07T23:39:18.906-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:39:18.906-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T23:39:18.915-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:39:18.915-0300] {dag.py:2594} INFO - Creating ORM DAG for twitter_dag
[2022-11-07T23:39:18.926-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:39:18.926-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-07T23:39:18.756169+00:00, run_after=2022-11-08T23:39:18.756169+00:00
[2022-11-07T23:39:18.966-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.229 seconds
[2022-11-07T23:39:29.733-0300] {processor.py:156} INFO - Started process (PID=967096) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:39:29.733-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:39:29.734-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:39:29.734-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:39:29.760-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:39:29.767-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:39:29.767-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T23:39:29.786-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:39:29.786-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-07T23:39:29.751147+00:00, run_after=2022-11-08T23:39:29.751147+00:00
[2022-11-07T23:39:29.810-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.080 seconds
[2022-11-07T23:40:11.050-0300] {processor.py:156} INFO - Started process (PID=967181) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:40:11.053-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:40:11.053-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:40:11.053-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:40:11.086-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:40:11.160-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:40:11.160-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T23:40:11.187-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:40:11.187-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-07T23:40:11.068702+00:00, run_after=2022-11-08T23:40:11.068702+00:00
[2022-11-07T23:40:11.236-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.189 seconds
[2022-11-07T23:40:50.384-0300] {processor.py:156} INFO - Started process (PID=967256) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:40:50.389-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:40:50.390-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:40:50.390-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:40:50.416-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:40:50.455-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:40:50.455-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T23:40:50.470-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:40:50.470-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-07T23:40:50.406667+00:00, run_after=2022-11-08T23:40:50.406667+00:00
[2022-11-07T23:40:50.495-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.115 seconds
[2022-11-07T23:41:24.155-0300] {processor.py:156} INFO - Started process (PID=967320) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:41:24.156-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:41:24.157-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:41:24.156-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:41:24.186-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:41:24.232-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:41:24.232-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T23:41:24.250-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:41:24.250-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-07T23:41:24.169966+00:00, run_after=2022-11-08T23:41:24.169966+00:00
[2022-11-07T23:41:24.277-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.127 seconds
[2022-11-07T23:41:54.702-0300] {processor.py:156} INFO - Started process (PID=967387) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:41:54.703-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:41:54.704-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:41:54.703-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:41:54.729-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:41:54.777-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:41:54.777-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T23:41:54.791-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:41:54.791-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-07T23:41:54.719142+00:00, run_after=2022-11-08T23:41:54.719142+00:00
[2022-11-07T23:41:54.851-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.156 seconds
[2022-11-07T23:42:28.186-0300] {processor.py:156} INFO - Started process (PID=967581) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:42:28.188-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:42:28.189-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:42:28.189-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:42:28.221-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:42:28.269-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:42:28.269-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T23:42:28.284-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:42:28.284-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-07T23:42:28.207903+00:00, run_after=2022-11-08T23:42:28.207903+00:00
[2022-11-07T23:42:28.311-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.129 seconds
[2022-11-07T23:43:07.224-0300] {processor.py:156} INFO - Started process (PID=967656) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:43:07.227-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-07T23:43:07.228-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:43:07.228-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:43:07.261-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-07T23:43:07.305-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:43:07.305-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-07T23:43:07.321-0300] {logging_mixin.py:117} INFO - [2022-11-07T23:43:07.320-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-07T23:43:07.249903+00:00, run_after=2022-11-08T23:43:07.249903+00:00
[2022-11-07T23:43:07.349-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.133 seconds
[2022-11-08T19:26:11.200-0300] {processor.py:156} INFO - Started process (PID=11751) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:26:11.201-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:26:11.202-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:26:11.202-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:26:11.221-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:26:11.218-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 5, in <module>
    from airflow.contrib.operators.spark_submit_operator import SparkSubmitOperator
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-11-08T19:26:11.222-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:26:11.247-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.050 seconds
[2022-11-08T19:26:51.636-0300] {processor.py:156} INFO - Started process (PID=11840) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:26:51.641-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:26:51.642-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:26:51.642-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:26:51.659-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:26:51.658-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 5, in <module>
    from airflow.contrib.operators.spark_submit_operator import SparkSubmitOperator
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-11-08T19:26:51.659-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:26:51.681-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.049 seconds
[2022-11-08T19:27:22.490-0300] {processor.py:156} INFO - Started process (PID=11902) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:27:22.494-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:27:22.495-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:27:22.495-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:27:22.525-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:27:22.523-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 5, in <module>
    from airflow.contrib.operators.spark_submit_operator import SparkSubmitOperator
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-11-08T19:27:22.526-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:27:22.564-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.080 seconds
[2022-11-08T19:28:00.000-0300] {processor.py:156} INFO - Started process (PID=11975) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:28:00.005-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:28:00.007-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:28:00.006-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:28:00.036-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:28:00.034-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 5, in <module>
    from airflow.contrib.operators.spark_submit_operator import SparkSubmitOperator
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-11-08T19:28:00.036-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:28:00.060-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.071 seconds
[2022-11-08T19:28:39.577-0300] {processor.py:156} INFO - Started process (PID=12180) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:28:39.579-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:28:39.580-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:28:39.580-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:28:39.596-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:28:39.595-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 5, in <module>
    from airflow.contrib.operators.spark_submit_operator import SparkSubmitOperator
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-11-08T19:28:39.596-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:28:39.616-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.043 seconds
[2022-11-08T19:29:13.763-0300] {processor.py:156} INFO - Started process (PID=12247) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:29:13.764-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:29:13.765-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:29:13.765-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:29:13.782-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:29:13.781-0300] {dagbag.py:330} ERROR - Failed to import: /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
Traceback (most recent call last):
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py", line 5, in <module>
    from airflow.contrib.operators.spark_submit_operator import SparkSubmitOperator
  File "/home/renato/Documentos/datapipeline/.env/lib/python3.10/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-11-08T19:29:13.783-0300] {processor.py:770} WARNING - No viable dags retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:29:13.810-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.049 seconds
[2022-11-08T19:30:58.022-0300] {processor.py:156} INFO - Started process (PID=12346) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:30:58.025-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:30:58.026-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:30:58.026-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:30:58.064-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:30:58.109-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:30:58.108-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:30:58.125-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:30:58.125-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:30:58.056312+00:00, run_after=2022-11-09T19:30:58.056312+00:00
[2022-11-08T19:30:58.159-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.140 seconds
[2022-11-08T19:31:40.270-0300] {processor.py:156} INFO - Started process (PID=12433) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:31:40.271-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:31:40.272-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:31:40.272-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:31:40.295-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:31:40.339-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:31:40.339-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:31:40.357-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:31:40.356-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:31:40.287025+00:00, run_after=2022-11-09T19:31:40.287025+00:00
[2022-11-08T19:31:40.390-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.122 seconds
[2022-11-08T19:32:18.157-0300] {processor.py:156} INFO - Started process (PID=12515) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:32:18.160-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:32:18.160-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:32:18.160-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:32:18.192-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:32:18.232-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:32:18.232-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:32:18.246-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:32:18.246-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:32:18.181894+00:00, run_after=2022-11-09T19:32:18.181894+00:00
[2022-11-08T19:32:18.271-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.117 seconds
[2022-11-08T19:32:55.906-0300] {processor.py:156} INFO - Started process (PID=12589) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:32:55.908-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:32:55.910-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:32:55.910-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:32:55.940-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:32:55.982-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:32:55.982-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:32:55.997-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:32:55.997-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:32:55.931481+00:00, run_after=2022-11-09T19:32:55.931481+00:00
[2022-11-08T19:32:56.024-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.131 seconds
[2022-11-08T19:33:27.705-0300] {processor.py:156} INFO - Started process (PID=12781) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:33:27.706-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:33:27.707-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:33:27.706-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:33:27.735-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:33:27.775-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:33:27.775-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:33:27.790-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:33:27.790-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:33:27.725412+00:00, run_after=2022-11-09T19:33:27.725412+00:00
[2022-11-08T19:33:27.825-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.123 seconds
[2022-11-08T19:33:59.948-0300] {processor.py:156} INFO - Started process (PID=12989) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:33:59.951-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:33:59.952-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:33:59.952-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:33:59.999-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:34:00.060-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:34:00.060-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:34:00.076-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:34:00.076-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:33:59.984533+00:00, run_after=2022-11-09T19:33:59.984533+00:00
[2022-11-08T19:34:00.116-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.171 seconds
[2022-11-08T19:34:31.539-0300] {processor.py:156} INFO - Started process (PID=13149) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:34:31.539-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:34:31.540-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:34:31.540-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:34:31.566-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:34:31.605-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:34:31.605-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:34:31.624-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:34:31.624-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:34:31.557208+00:00, run_after=2022-11-09T19:34:31.557208+00:00
[2022-11-08T19:34:31.653-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.117 seconds
[2022-11-08T19:35:05.718-0300] {processor.py:156} INFO - Started process (PID=13234) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:35:05.720-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:35:05.721-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:35:05.721-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:35:05.750-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:35:05.787-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:35:05.787-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:35:05.802-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:35:05.802-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:35:05.742669+00:00, run_after=2022-11-09T19:35:05.742669+00:00
[2022-11-08T19:35:05.825-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.111 seconds
[2022-11-08T19:35:36.978-0300] {processor.py:156} INFO - Started process (PID=13297) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:35:36.981-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:35:36.982-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:35:36.981-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:35:37.013-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:35:37.055-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:35:37.055-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:35:37.072-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:35:37.071-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:35:37.006096+00:00, run_after=2022-11-09T19:35:37.006096+00:00
[2022-11-08T19:35:37.105-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.132 seconds
[2022-11-08T19:36:13.717-0300] {processor.py:156} INFO - Started process (PID=13373) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:36:13.719-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:36:13.719-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:36:13.719-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:36:13.744-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:36:13.786-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:36:13.785-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:36:13.801-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:36:13.801-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:36:13.735874+00:00, run_after=2022-11-09T19:36:13.735874+00:00
[2022-11-08T19:36:13.828-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.114 seconds
[2022-11-08T19:36:52.724-0300] {processor.py:156} INFO - Started process (PID=13481) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:36:52.729-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:36:52.730-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:36:52.729-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:36:52.760-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:36:52.809-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:36:52.809-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:36:52.828-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:36:52.827-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:36:52.751462+00:00, run_after=2022-11-09T19:36:52.751462+00:00
[2022-11-08T19:36:52.895-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.173 seconds
[2022-11-08T19:37:28.681-0300] {processor.py:156} INFO - Started process (PID=13591) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:37:28.685-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:37:28.687-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:37:28.686-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:37:28.722-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:37:28.763-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:37:28.762-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:37:28.778-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:37:28.778-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:37:28.710872+00:00, run_after=2022-11-09T19:37:28.710872+00:00
[2022-11-08T19:37:28.806-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.134 seconds
[2022-11-08T19:38:08.199-0300] {processor.py:156} INFO - Started process (PID=13677) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:38:08.201-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:38:08.201-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:38:08.201-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:38:08.264-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:38:08.346-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:38:08.346-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:38:08.387-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:38:08.387-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:38:08.247862+00:00, run_after=2022-11-09T19:38:08.247862+00:00
[2022-11-08T19:38:08.425-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.229 seconds
[2022-11-08T19:38:45.835-0300] {processor.py:156} INFO - Started process (PID=14009) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:38:45.838-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:38:45.840-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:38:45.840-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:38:45.881-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:38:45.919-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:38:45.919-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:38:45.934-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:38:45.934-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:38:45.870356+00:00, run_after=2022-11-09T19:38:45.870356+00:00
[2022-11-08T19:38:45.958-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.134 seconds
[2022-11-08T19:39:18.133-0300] {processor.py:156} INFO - Started process (PID=14072) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:39:18.135-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:39:18.136-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:39:18.135-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:39:18.167-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:39:18.207-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:39:18.207-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:39:18.222-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:39:18.222-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:39:18.158202+00:00, run_after=2022-11-09T19:39:18.158202+00:00
[2022-11-08T19:39:18.246-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.117 seconds
[2022-11-08T19:39:50.355-0300] {processor.py:156} INFO - Started process (PID=14135) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:39:50.356-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:39:50.357-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:39:50.356-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:39:50.387-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:39:50.424-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:39:50.424-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:39:50.440-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:39:50.440-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:39:50.379757+00:00, run_after=2022-11-09T19:39:50.379757+00:00
[2022-11-08T19:39:50.463-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.113 seconds
[2022-11-08T19:40:21.695-0300] {processor.py:156} INFO - Started process (PID=14204) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:40:21.698-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:40:21.699-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:40:21.699-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:40:21.729-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:40:21.768-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:40:21.768-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:40:21.782-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:40:21.782-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:40:21.718908+00:00, run_after=2022-11-09T19:40:21.718908+00:00
[2022-11-08T19:40:21.805-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.117 seconds
[2022-11-08T19:40:56.639-0300] {processor.py:156} INFO - Started process (PID=14436) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:40:56.646-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:40:56.650-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:40:56.650-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:40:56.673-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:40:56.713-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:40:56.713-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:40:56.729-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:40:56.729-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:40:56.665775+00:00, run_after=2022-11-09T19:40:56.665775+00:00
[2022-11-08T19:40:56.758-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.123 seconds
[2022-11-08T19:41:27.705-0300] {processor.py:156} INFO - Started process (PID=14497) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:41:27.705-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:41:27.706-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:41:27.706-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:41:27.729-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:41:27.774-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:41:27.773-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:41:27.790-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:41:27.790-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:41:27.720893+00:00, run_after=2022-11-09T19:41:27.720893+00:00
[2022-11-08T19:41:27.823-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.121 seconds
[2022-11-08T19:42:02.312-0300] {processor.py:156} INFO - Started process (PID=14792) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:42:02.316-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:42:02.316-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:42:02.316-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:42:02.342-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:42:02.383-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:42:02.383-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:42:02.398-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:42:02.398-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:42:02.334231+00:00, run_after=2022-11-09T19:42:02.334231+00:00
[2022-11-08T19:42:02.422-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.118 seconds
[2022-11-08T19:42:41.204-0300] {processor.py:156} INFO - Started process (PID=14883) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:42:41.206-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:42:41.207-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:42:41.207-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:42:41.231-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:42:41.272-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:42:41.271-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:42:41.286-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:42:41.286-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:42:41.223462+00:00, run_after=2022-11-09T19:42:41.223462+00:00
[2022-11-08T19:42:41.311-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.109 seconds
[2022-11-08T19:43:17.498-0300] {processor.py:156} INFO - Started process (PID=15091) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:43:17.511-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:43:17.511-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:43:17.511-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:43:17.535-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:43:17.573-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:43:17.573-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:43:17.588-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:43:17.588-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:43:17.526713+00:00, run_after=2022-11-09T19:43:17.526713+00:00
[2022-11-08T19:43:17.611-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.116 seconds
[2022-11-08T19:43:49.114-0300] {processor.py:156} INFO - Started process (PID=15154) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:43:49.116-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:43:49.117-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:43:49.117-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:43:49.154-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:43:49.192-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:43:49.191-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:43:49.207-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:43:49.207-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:43:49.146311+00:00, run_after=2022-11-09T19:43:49.146311+00:00
[2022-11-08T19:43:49.242-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.133 seconds
[2022-11-08T19:44:25.079-0300] {processor.py:156} INFO - Started process (PID=15236) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:44:25.081-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:44:25.081-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:44:25.081-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:44:25.120-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:44:25.179-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:44:25.179-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:44:25.202-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:44:25.202-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:44:25.109781+00:00, run_after=2022-11-09T19:44:25.109781+00:00
[2022-11-08T19:44:25.239-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.167 seconds
[2022-11-08T19:44:57.584-0300] {processor.py:156} INFO - Started process (PID=15298) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:44:57.586-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:44:57.588-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:44:57.587-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:44:57.641-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:44:57.699-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:44:57.699-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:44:57.714-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:44:57.714-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:44:57.622891+00:00, run_after=2022-11-09T19:44:57.622891+00:00
[2022-11-08T19:44:57.739-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.160 seconds
[2022-11-08T19:45:44.390-0300] {processor.py:156} INFO - Started process (PID=15647) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:45:44.393-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:45:44.394-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:45:44.394-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:45:44.422-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:45:44.468-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:45:44.468-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:45:44.483-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:45:44.483-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:45:44.414741+00:00, run_after=2022-11-09T19:45:44.414741+00:00
[2022-11-08T19:45:44.511-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.128 seconds
[2022-11-08T19:46:19.331-0300] {processor.py:156} INFO - Started process (PID=15729) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:46:19.333-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:46:19.334-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:46:19.334-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:46:19.358-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:46:19.397-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:46:19.397-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:46:19.411-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:46:19.411-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:46:19.350751+00:00, run_after=2022-11-09T19:46:19.350751+00:00
[2022-11-08T19:46:19.433-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.105 seconds
[2022-11-08T19:46:55.130-0300] {processor.py:156} INFO - Started process (PID=15803) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:46:55.132-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:46:55.133-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:46:55.132-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:46:55.159-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:46:55.199-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:46:55.199-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:46:55.213-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:46:55.213-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:46:55.151698+00:00, run_after=2022-11-09T19:46:55.151698+00:00
[2022-11-08T19:46:55.237-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.112 seconds
[2022-11-08T19:47:48.044-0300] {processor.py:156} INFO - Started process (PID=16153) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:47:48.046-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:47:48.047-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:47:48.047-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:47:48.072-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:47:48.115-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:47:48.114-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:47:48.130-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:47:48.129-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:47:48.064865+00:00, run_after=2022-11-09T19:47:48.064865+00:00
[2022-11-08T19:47:48.154-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.113 seconds
[2022-11-08T19:47:56.171-0300] {processor.py:156} INFO - Started process (PID=16167) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:47:56.172-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:47:56.173-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:47:56.173-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:47:56.197-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:47:56.204-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:47:56.204-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:47:56.222-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:47:56.222-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:47:56.190086+00:00, run_after=2022-11-09T19:47:56.190086+00:00
[2022-11-08T19:47:56.244-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.076 seconds
[2022-11-08T19:48:34.860-0300] {processor.py:156} INFO - Started process (PID=16394) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:48:34.863-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:48:34.864-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:48:34.864-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:48:34.902-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:48:34.949-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:48:34.949-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:48:34.963-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:48:34.963-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:48:34.889186+00:00, run_after=2022-11-09T19:48:34.889186+00:00
[2022-11-08T19:48:34.986-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.134 seconds
[2022-11-08T19:48:42.216-0300] {processor.py:156} INFO - Started process (PID=16408) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:48:42.218-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:48:42.219-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:48:42.219-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:48:42.313-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:48:42.332-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:48:42.331-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:48:42.364-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:48:42.364-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:48:42.286948+00:00, run_after=2022-11-09T19:48:42.286948+00:00
[2022-11-08T19:48:42.398-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.198 seconds
[2022-11-08T19:49:49.620-0300] {processor.py:156} INFO - Started process (PID=16798) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:49:49.622-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:49:49.623-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:49:49.623-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:49:49.654-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:49:49.694-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:49:49.694-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:49:49.708-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:49:49.708-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:49:49.644612+00:00, run_after=2022-11-09T19:49:49.644612+00:00
[2022-11-08T19:49:49.737-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.121 seconds
[2022-11-08T19:49:58.202-0300] {processor.py:156} INFO - Started process (PID=16817) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:49:58.203-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:49:58.204-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:49:58.203-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:49:58.232-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:49:58.238-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:49:58.238-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:49:58.257-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:49:58.256-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:49:58.224481+00:00, run_after=2022-11-09T19:49:58.224481+00:00
[2022-11-08T19:49:58.278-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.079 seconds
[2022-11-08T19:52:00.773-0300] {processor.py:156} INFO - Started process (PID=17768) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:52:00.777-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:52:00.778-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:52:00.778-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:52:00.804-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:52:00.854-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:52:00.854-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:52:00.872-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:52:00.872-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:52:00.795301+00:00, run_after=2022-11-09T19:52:00.795301+00:00
[2022-11-08T19:52:00.917-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.147 seconds
[2022-11-08T19:52:33.200-0300] {processor.py:156} INFO - Started process (PID=17831) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:52:33.202-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:52:33.203-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:52:33.203-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:52:33.225-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:52:33.267-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:52:33.267-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:52:33.282-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:52:33.282-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:52:33.217312+00:00, run_after=2022-11-09T19:52:33.217312+00:00
[2022-11-08T19:52:33.305-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.107 seconds
[2022-11-08T19:53:07.357-0300] {processor.py:156} INFO - Started process (PID=17895) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:53:07.358-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:53:07.358-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:53:07.358-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:53:07.381-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:53:07.439-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:53:07.439-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:53:07.456-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:53:07.455-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:53:07.373760+00:00, run_after=2022-11-09T19:53:07.373760+00:00
[2022-11-08T19:53:07.504-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.150 seconds
[2022-11-08T19:53:43.219-0300] {processor.py:156} INFO - Started process (PID=18092) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:53:43.222-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:53:43.222-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:53:43.222-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:53:43.245-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:53:43.290-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:53:43.290-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:53:43.305-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:53:43.305-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:53:43.236610+00:00, run_after=2022-11-09T19:53:43.236610+00:00
[2022-11-08T19:53:43.331-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.114 seconds
[2022-11-08T19:54:22.490-0300] {processor.py:156} INFO - Started process (PID=18168) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:54:22.492-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:54:22.492-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:54:22.492-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:54:22.515-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:54:22.555-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:54:22.555-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:54:22.570-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:54:22.570-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:54:22.506993+00:00, run_after=2022-11-09T19:54:22.506993+00:00
[2022-11-08T19:54:22.597-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.110 seconds
[2022-11-08T19:54:54.845-0300] {processor.py:156} INFO - Started process (PID=18233) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:54:54.845-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:54:54.846-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:54:54.846-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:54:54.868-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:54:54.905-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:54:54.905-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:54:54.921-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:54:54.921-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:54:54.860271+00:00, run_after=2022-11-09T19:54:54.860271+00:00
[2022-11-08T19:54:54.947-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.105 seconds
[2022-11-08T19:55:27.695-0300] {processor.py:156} INFO - Started process (PID=18321) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:55:27.697-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:55:27.698-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:55:27.698-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:55:27.723-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:55:27.768-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:55:27.768-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:55:27.787-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:55:27.787-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:55:27.714612+00:00, run_after=2022-11-09T19:55:27.714612+00:00
[2022-11-08T19:55:27.821-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.128 seconds
[2022-11-08T19:56:00.269-0300] {processor.py:156} INFO - Started process (PID=18386) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:56:00.269-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:56:00.271-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:56:00.271-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:56:00.300-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:56:00.360-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:56:00.359-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:56:00.376-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:56:00.376-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:56:00.289436+00:00, run_after=2022-11-09T19:56:00.289436+00:00
[2022-11-08T19:56:00.401-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.135 seconds
[2022-11-08T19:56:36.860-0300] {processor.py:156} INFO - Started process (PID=18459) to work on /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:56:36.863-0300] {processor.py:758} INFO - Processing file /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py for tasks to queue
[2022-11-08T19:56:36.863-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:56:36.863-0300] {dagbag.py:525} INFO - Filling up the DagBag from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:56:36.888-0300] {processor.py:768} INFO - DAG(s) dict_keys(['twitter_dag']) retrieved from /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py
[2022-11-08T19:56:36.931-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:56:36.930-0300] {dag.py:2573} INFO - Sync 1 DAGs
[2022-11-08T19:56:36.947-0300] {logging_mixin.py:117} INFO - [2022-11-08T19:56:36.946-0300] {dag.py:3324} INFO - Setting next_dagrun for twitter_dag to 2022-11-08T19:56:36.879083+00:00, run_after=2022-11-09T19:56:36.879083+00:00
[2022-11-08T19:56:36.972-0300] {processor.py:178} INFO - Processing /home/renato/Documentos/datapipeline/airflow/dags/twitter_dag.py took 0.115 seconds
